speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
```

Comment:
I didn't remove the intent, just the samples as they are not required

> To implement a standard built-in intent, include the intent in your intent schema and then add handling for the intent in your code. You do not need to provide any sample utterances for these intents, although you can if you want to extend the intent.

https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/built-in-intent-ref/standard-intents#about-canceling-and-stopping